src/Core/Maintenance/SalesChannel/Command/SalesChannelReplaceUrlCommand.php

```diff
@@ -0,0 +1,105 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Maintenance\SalesChannel\Command;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\Aggregate\SalesChannelDomain\SalesChannelDomainCollection;
+use Shopware\Core\System\SalesChannel\Aggregate\SalesChannelDomain\SalesChannelDomainEntity;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Validator\Constraints\NotEqualTo;
+use Symfony\Component\Validator\Constraints\Url;
+use Symfony\Component\Validator\Validation;
+
+/**
+ * @internal
+ */
+#[AsCommand(
+    name: 'sales-channel:replace:url',
+    description: 'Replaces the URL of a sales channel with a new URL',
+)]
+#[Package('discovery')]
+class SalesChannelReplaceUrlCommand extends Command
+{
+    /**
+     * @internal
```

Comment:
The construct does not need to be marked as internal if the whole class is already internal :)

Reply:
removed